docs/configuration/integrations/integrations-next/app-agent-receiver-config.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "app_agent_config"
++++
+
+# app_agent_receiver_config
+
+The `app_agent_receiver_config` block configures the `app-agent-receiver`
```

Comment:
Should this use underscores since that's the name of the integration you'd specify in the config file?

```suggestion
The `app_agent_receiver_config` block configures the `app_agent_receiver`
```